src/representer/mapping.nim

```diff
@@ -0,0 +1,45 @@
+import hashes, tables, strutils
+
+type
+  NormalizedIdent = string
```

Comment:
```suggestion
  NormalizedIdent = distinct string
```
Is this route better? Of course, then we borrow/define some operators as needed.

Reply:
As above, I just wanted to be able to use `string` procs and funcs. What do you see as being the advantages of `distinct`?

Reply:
As above, I have ended up going with this solution in [00eb4d8](https://github.com/exercism/nim-representer/pull/3/commits/00eb4d82b894f981617c3e09bd8a0a67aaf20b65)